site2/docs/administration-proxy.md

```diff
@@ -14,8 +14,8 @@ Before using the proxy, you need to configure it with the brokers addresses in t
 
 Pulsar uses [ZooKeeper](https://zookeeper.apache.org) for service discovery. To connect the proxy to ZooKeeper, specify the following in `conf/proxy.conf`.
 ```properties
-zookeeperServers=zk-0,zk-1,zk-2
-configurationStoreServers=zk-0:2184,zk-remote:2184
+metadataStoreUrl=my-zk-0:2181,my-zk-1:2181,my-zk-2:2181
+configurationMetadataStoreUrl=my-zk-0:2184,my-zk-remote:2184
```

Comment:
There is no need to change this document.

Reply:
Once #13777 will be merged to master, you can ignore this review.

Reply:
> Once #13777 will be merged to master, you can ignore this review.

Agree. Thanks for pointing this out.

Reply:
Hi @RobertIndie, can you pls help review this modification to resolve conflict with PR#14081? Thanks.